.github/workflows/quick-merge.yml

```
@@ -0,0 +1,25 @@
+name: Quick Merge
+
+on:
+  pull_request_review:
+    types: [submitted]
+    branches: [main, develop]
```

actionlint (1.7.7)

"branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)